src/main/java/com/aerospike/mapper/tools/utils/TypeUtils.java

```diff
@@ -229,7 +229,11 @@ private static TypeMapper getMapper(Class<?> clazz, AnnotatedType type, IBaseAer
                         } else {
                             // Reference
                             ReferenceConfig ref = binConfig.getReference();
-                            typeMapper = new ObjectReferenceMapper(ClassCache.getInstance().loadClass(clazz, mapper), ref.getLazy(), ref.getBatchLoad(), ref.getType(), mapper);
+                            typeMapper = new ObjectReferenceMapper(
+                                    ClassCache.getInstance().loadClass(clazz, mapper),
+                                    ref.getLazy() == null ? false : ref.getLazy(),
```

Comment:
After consulting with Tim, we can improve it but at some point we will need a refactor of yaml/annotation config reading which will affect this place as well - I'm ok with the scope of this PR just because its only a single place of null check but later we will probably want to refactor/improve/clean some areas of the Java Object Mapper including this specific flow